src/mon/OSDMonitor.cc

```diff
@@ -4764,9 +4764,16 @@ void OSDMonitor::check_osdmap_subs()
 void OSDMonitor::check_osdmap_sub(Subscription *sub)
 {
   dout(10) << __func__ << " " << sub << " next " << sub->next
-	   << (sub->onetime ? " (onetime)":" (ongoing)") << dendl;
+	   << (sub->onetime ? " (onetime)":" (ongoing)")
+	   << (sub->want_latest_osdmap ? " (noosd)":" (osd)")
```

Comment:
Why "noosd" and "osd" when the flag is named `CEPH_SUBSCRIBE_LATEST_OSDMAP`?

Reply:
I think incremental mapping isn't necessary for non OSD services.

Reply:
it's better to directly name it  as 'latest'